src/backend/cuda/CMakeLists.txt

```diff
@@ -314,6 +316,47 @@ CL_KERNEL_TO_H(
     EOF "1"
     )
 
+SET(libdevice_bc "")
+IF (USE_LIBDEVICE)
+  SET(libdevice_computes "")
+  LIST(APPEND libdevice_computes "20" "30" "35" "50")
```

Comment:
Use CMake LIST `COMPUTES_DETECTED_LIST` here. This includes both detected computes as well as manually set computes.